### PR TITLE
Development feat#65

### PR DESCRIPTION
**Pull request by tomski80**
User can only add one empty card. If try to add more, then will see the message to edit empty card before adding more cards. If last card is left empty and user navigate away from edit component this card will be removed. (so there will not be empty cards in collection).

### CODE REVIEW

File: src/components/CollectionEdit.vue [hunk 1]
```diff
@@ -19,6 +19,7 @@
     class="newCard"
     @click="add"
   >+</button>
+  <p v-if=emptyWarningMessage>Please edit <em>Question</em> and <em>Answer</em> before adding new Card</p>
```
Comment (AntonTymczenko) [inline src/components/CollectionEdit.vue]: v-if="..."
you've missed double quotes around the variable

**Comment by AntonTymczenko** [comment]
oh, there's still one bug. 
when user makes an empty field in some card and then deletes this card -- an error is still in "errors" array. 

so we should remove errors with this index when deleting a card

**Comment by tomski80** [comment]
I think you over-engineered this now. It doesn't show the message now if I try to add another empty card that you have to fill in q & a before adding new empty - and in my opinion it is confusing, because button suddenly doesn't work . Also message to fill question and answer show when I start to writing either question or answer - so it is no longer empty (have at least one character in it) - so user obviously already doing it. Which is confusing as well. I think we need to rethink What we trying to communicate to the user here.

**Comment by svmi3195** [comment]
You make a code for testing if the card is empty
`    lastCardIsEmpty () {`
`      const lastCard = this.collection.items[this.lastIndex]`
`      return lastCard.q === '' && lastCard.a === ''`
`    }`
but you don't use it. I think you could use it here:
`    removeLastEmpty(){`
`      if(this.collection.items[this.lastIndex].q === ''`
`        && this.collection.items[this.lastIndex].a === '')`
`      {`
`        this.collection.items.pop()`
`      }`
`    }`

**Comment by AntonTymczenko** [comment]
@svmi3195 nicely noticed! I've put this computed property here too!

**Comment by AntonTymczenko** [comment]
@tomski80 you are right, it is still buggy, but we can improve it :D I'll think about the solution in a few nearest hours...

**Review by AntonTymczenko — APPROVED** [review]

**Pull request merged**